ultravox/data/datasets.py

```diff
@@ -8,70 +8,117 @@
 import os
 import tempfile
 import warnings
-from enum import Enum
-from typing import Any, Callable, Dict, List, Optional, Sequence
+from typing import Any, Dict, List, Optional, Sequence
 
-import datasets
+import datasets as hf_datasets
 import jinja2
 import librosa
 import numpy as np
-import requests
 import soundfile as sf
 import streaming as mds
 import torch
 import torch.nn.functional as F
 import transformers
+from simple_parsing import helpers
 from torch.utils import data
 
-from ultravox.data import dataset_config
 from ultravox.data import text_proc
 
 SAMPLE_RATE = 16000
 
-TRANSCRIBE_INPUT_TASK = "transcribe_input"
-TRANSCRIBE_OUTPUT_TASK = "transcribe_output"
-ANSWER_TASK = "answer"
-
-TRANSCRIBE_PROMPTS = [
-    # from Gazelle
-    "Transcribe\n<|audio|>",
-    "Transcribe exactly what is said here\n<|audio|>",
-    "Repeat exactly what is written here: <|audio|>",
-    "Write exactly what was said: <|audio|>",
-    "First listen to the clip. Then, transcribe exactly what is said. <|audio|>",
-    # from https://arxiv.org/pdf/2402.08846
-    "Transcribe speech to text: <|audio|>",
-    # from GPT-4
-    "Capture every word from the audio verbatim\n<|audio|>",
-    "Convert speech to text from audio\n<|audio|>",
-    "Listen and transcribe the complete text from audio\n<|audio|>",
-    "Record in writing what is spoken in audio\n<|audio|>",
-    "Transcribe the spoken words from audio with exact wording and punctuation\n<|audio|>",
-]
```

Comment:
I know we haven't been using these, but just making sure: @zqhuang211 did you do an experiment to show that randomizing the prompt is not useful at all?

Reply:
We could bring this functionality back by letting `user_template` be a List, if needed.

Reply:
Agreed. I don't want to block on this.